packages/components/src/components/date-of-birth-picker/date-of-birth-picker.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { type ReactElement } from 'react';
+import { toMoment } from '@deriv/shared';
+import DatePicker from '../date-picker';
+
+type TDateOfBirthPicker = {
```

Comment:
@amir-deriv since all the props are passed directly to the child <DatePicker and not used in this component, we don't have to declare this type right here. Instead, we can declare the props type once for the child <DatePicker in its own file, export it from there and import it into this file like this 🙂 :
`import DatePicker, { TDatePickerProps } from '../date-picker';`

Reply:
Done